<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtSpec</class>
 <widget class="QMainWindow" name="QtSpec">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtSpec</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="labStr">
         <property name="text">
          <string>字符串</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="editStr"/>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="btnStr">
         <property name="text">
          <string>字符Unicode</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labChar">
         <property name="text">
          <string>单个字符</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="editChar"/>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="btnChar">
         <property name="text">
          <string>单个字符特性</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>其他测试和功能</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QPushButton" name="btnQChar">
         <property name="text">
          <string>QChar比较和替换</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="btnLati">
         <property name="text">
          <string>与Lati1转换</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="btnClear">
         <property name="text">
          <string>清空文本框</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btnUtf16">
         <property name="text">
          <string>与UTF-16转换</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>显示结果</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QPlainTextEdit" name="txtShow"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>QChar特性判断</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QCheckBox" name="chkDigit">
           <property name="text">
            <string>isDigit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkLetter">
           <property name="text">
            <string>isLetter</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkLetOrNum">
           <property name="text">
            <string>isLetterOrNumber</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkUpper">
           <property name="text">
            <string>isUpper</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkLower">
           <property name="text">
            <string>isLower</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkMark">
           <property name="text">
            <string>isMark</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkSpace">
           <property name="text">
            <string>isSpace</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkSymbol">
           <property name="text">
            <string>isSymbol</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkPunct">
           <property name="text">
            <string>isPunct</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>612</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuQChar">
    <property name="title">
     <string>QChar</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <addaction name="menuQChar"/>
   <addaction name="menuAbout"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
